src/Toolkit/Toolkit/UI/Controls/SymbolDisplay/SymbolDisplay.cs

```diff
@@ -67,33 +68,39 @@ private void OnSymbolChanged(Symbology.Symbol? oldValue, Symbology.Symbol? newVa
                 {
                     OnEventAction = (instance, source, eventArgs) =>
                     {
-                        Refresh();
+                        _ = Refresh();
```

Comment:
Since you're not awaiting the refresh anywhere, there's not much point to this change. You're still creating a potentially unobserved task exception.
async-void is ok, as long as there's a try-catch to catch any exceptions. Adding a try-catch should be sufficient.